cinema/serializers.py

```diff
@@ -1 +1,101 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (
+    CinemaHall,
+    Genre,
+    Actor,
+    Movie,
+    MovieSession,
+    Order,
+    Ticket
+)
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = CinemaHall
+        fields = ("id", "name", "rows", "seats_in_row", "capacity")
```

Comment:
Adding a comma at the end of the last item in a tuple or list is a common practice in Python, fix in all places